scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -1,5 +1,5 @@
 #
-# Case sample workflow for a single pair of case-control samples.  Includes GATK CNV and ACNV.
+# Case sample workflow for a list of pairs of case-control samples.  Includes GATK CNV and ACNV. Supports both WGS and exome capture samples.
```

Comment:
Maybe "WGS and exome capture" -> "WGS and targeted WES" here, and then just "WES" below for simplicity?  You refer to the workflow as both "exome capture" and "capture" below.